### ISSUE TITLE
WebStorm hangs up on start, after CodeStream installation

### ISSUE THREAD

**Issue by wnukwiktor** [TeamCodeStream/codestream#242]
CodeStream version: **8.3.2+84**

WebStorm-**202.6397.88**

Ubuntu 20.04.1 LTS

Problem:
After installing CodeStream through plugins section in WebStorm, it ask to restart IDE. After restart, however, i**t prevents the project from loading - it gets stuck on** _'Reopening files...'_, and a tiny text "_CodeStream: Sign In..."_ appears in the bottom right corner. See attached screenshot.

The only way to close WebStorm is by killing its process from terminal.

After trying to open WebStorm from terminal, using:
` /opt/WebStorm-202.6397.88/bin$ sh webstorm.sh `

I get the following start logs warning: **Unsupported notification method: codestream/didStartLogin**:
```
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
2020-08-14 08:58:25,051 [   4542]   WARN - .codestream.agent.AgentService - (node:7996) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead. 
Aug 14, 2020 8:58:25 AM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
WARNING: Unsupported notification method: codestream/didStartLogin
```
The only way to make WebStorm start and load the project is by removing _codestream-jb-jcef_ folder from _~/.local/share/JetBrains/WebStorm2020.2_ 

![Screenshot from 2020-08-10 10-00-22](https://user-images.githubusercontent.com/13747783/90225721-1917f700-de12-11ea-839c-b20591c37ebb.png)

**Comment by planteater** [comment]
Thanks for the report @wnukwiktor ! We'll dig into this asap.

**Comment by planteater** [comment]
@wnukwiktor - As a workaround for now, go to Preferences > Tools > CodeStream and uncheck the "Use JCEF Webview" box, then restart your IDE.

**Comment by planteater** [comment]
@wnukwiktor - One more thing. Could I trouble you for a log file? In WebStorm, go to Help > Collect Logs and Diagnostic Data. This will open the finder where you should see a newly created zip file. Feel free to email it to me at support@codestream.com if you'd prefer not to post it here. Thanks!

**Comment by planteater** [comment]
@wnukwiktor - Just wanted to follow up again to see if there was any chance we could get that log file. Thanks!

**Comment by AndrewMorsillo** [comment]
@planteater Here's the Jetbrains Youtrack issue for this problem https://youtrack.jetbrains.com/issue/WEB-46679

Hopefully we can get this sorted out quickly, it's a real blocker to work. It even caused me to lose some work when a BSOD corrupted some files and I had to delete .idea to get webstorm to start again which removed local history I could have recovered the data from =(

**Comment by planteater** [comment]
@AndrewMorsillo - Sounds like the issue might be with WebStorm and not CodeStream, but I think we'll need a log file in order to determine that definitively. 

In WebStorm, go to Help > Collect Logs and Diagnostic Data. This will open the finder where you should see a newly created zip file. Feel free to email it to me at support@codestream.com if you'd prefer not to post it here. Thanks!

**Comment by planteater** [comment]
@AndrewMorsillo - Just wanted to follow up on that log file. Thanks!

**Comment by AndrewMorsillo** [comment]
Haven’t had a chance yet, sorry, been very busy and haven’t had time to reenable codestream and deal with the loss of webstorm settings from the crash. I’ll try to get it to you soon!

**Comment by masi** [comment]
It's only the auto-login that makes the troubles. If you disable it, PhpStorm starts up fine.
Another workaround ist to kill the network before starting PhpStorm and bring it up later again. Alternatively you can set temporarily api.codestream.com to 127.0.0.1 while PhpStorm starts.

**Comment by planteater** [comment]
@masi - Would you mind sending a log file as well? Thanks!

**Comment by lots0logs** [comment]
@planteater Looks like this issue appears in all JetBrains IDEs. However, I have not had any problems with data corruption. That sounds like it could be a separate issue or maybe OS specific. Anyway, here are the logs. I also submitted them via the IDE's crash report tool.

[threadDumps-freeze-20200828-141237-PS-202.6948.54.zip](https://github.com/TeamCodeStream/codestream/files/5143811/threadDumps-freeze-20200828-141237-PS-202.6948.54.zip)

**Comment by planteater** [comment]
Fixed in version 8.3.7 of the CodeStream extension.

**Comment by masi** [comment]
Auto-Login with JCEF works for me now.
